docs/pages/yaml-reference.md

```diff
@@ -42,6 +42,19 @@ This documentation effectively reflects the Java object tree under [RawServiceSp
 * `pods`
 
   This section contains a listing of all pod types managed by the service.
+  
+  * `ipc-mode`
+    IPC (POSIX/SysV IPC) namespace provides separation of named shared memory segments, semaphores and message queues. Shared memory segments are used to accelerate inter-process communication at memory speed, rather than through pipes or through the network stack.
+
+    Specifies the IPC mode to run pod in. Only PRIVATE is valid at the pod level.
```

Comment:
pod's ipc mode can only be private if this dc/os has disallowed sharing. https://github.com/dcos/dcos/blob/32ad0f7d053d6e107cc3a0fdd313dc50bc05ab62/gen/dcos-config.yaml#L433

otherwise, pod could still share host's ipc namespace.